src/yunohost/app.py

```diff
@@ -2574,6 +2576,31 @@ def unstable_apps():
     return output
 
 
+def _check_services_status_for_app(services):
+
+    logger.debug("Checking that required services are up and running...")
+
+    # Some apps use php-fpm or php5-fpm which is now php7.0-fpm
+    def replace_alias(service):
+        if service in ["php-fpm", "php5-fpm"]:
+            return "php7.0-fpm"
+        else:
+            return service
+    services = [replace_alias(s) for s in services]
+
+    # We only check those, mostly to ignore "custom" services
```

Comment:
Shouldn't we only checked already known services only instead? I'm not sure. That will also filtered out not already installed services and, maybe?, could prevent other errors.